<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAddTrack</class>
 <widget class="QWidget" name="FormAddTrack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="ratting">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
	padding: 0 8px;
	background:white;
	border:10px;
	color:rgb(10,100,180);
border: 1px solid black;
border-color: rgb(165, 165, 165);
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>470</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 10pt &quot;Showcard Gothic&quot;;
	padding: 0 8px;

	color:rgb(10,100,180);
border: 1px solid black;
border-color: rgb(165, 165, 165);
}</string>
   </property>
   <property name="text">
    <string>Rate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddMusic">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>590</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;Showcard Gothic&quot;;
	padding: 0 8px;
	background:white;
	border:10px;
	color:rgb(10,100,180);
}</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusInsert">
   <property name="geometry">
    <rect>
     <x>383</x>
     <y>40</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 10pt &quot;Showcard Gothic&quot;;
	padding: 0 8px;

	color:rgb(10,100,180);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>590</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;Showcard Gothic&quot;;
	padding: 0 8px;
	background:white;
	border:10px;
	color:rgb(10,100,180);
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>1001</width>
     <height>721</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Image/sky_blue_partly_cloudy_sky_blue_azur_azure_bright_blue_light-1239095.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>521</width>
     <height>631</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Image/Hinh-Nen-Trang-10.webp</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 24pt &quot;Showcard Gothic&quot;;
	padding: 0 8px;

	color:rgb(10,100,180);
}</string>
   </property>
   <property name="text">
    <string>ADDTrack</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 10pt &quot;Showcard Gothic&quot;;
	padding: 0 8px;

	color:rgb(10,100,180);

}</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="musicName">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{

	border: 1px solid black;
	padding: 0 8px;
	background:white;
	border-color: rgb(165, 165, 165);
	color:rgb(10,100,180);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorMusicName">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 6pt &quot;Segoergb(255, 21, 72) UI&quot;;
	
	color: rgb(191, 0, 0);
	
	padding: 0 8px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorMusicUrl">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 6pt &quot;Segoergb(255, 21, 72) UI&quot;;
	
	color: rgb(191, 0, 0);
	
	padding: 0 8px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMusicUrl">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>255</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 9pt &quot;Showcard Gothic&quot;;
	padding: 0 8px;
	background:white;
	border:10px;
	color:rgb(10,100,180);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/folder.png</normaloff>img/folder.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 10pt &quot;Showcard Gothic&quot;;
	padding: 0 8px;

	color:rgb(10,100,180);

}</string>
   </property>
   <property name="text">
    <string>Mucsic</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorMusicImg">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 6pt &quot;Segoergb(255, 21, 72) UI&quot;;
	
	color: rgb(191, 0, 0);
	
	padding: 0 8px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 10pt &quot;Showcard Gothic&quot;;
	padding: 0 8px;

	color:rgb(10,100,180);

}</string>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnImgUrl">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>345</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 9pt &quot;Showcard Gothic&quot;;
	padding: 0 8px;
	background:white;
	border:10px;
	color:rgb(10,100,180);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/folder.png</normaloff>img/folder.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 10pt &quot;Showcard Gothic&quot;;
	padding: 0 8px;

	color:rgb(10,100,180);
border: 1px solid black;
border-color: rgb(165, 165, 165);
}</string>
   </property>
   <property name="text">
    <string>Category:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="category">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{


	padding: 0 8px;
	background:white;
	color:rgb(10,100,180);
border: 1px solid black;
border-color: rgb(165, 165, 165);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>530</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 10pt &quot;Showcard Gothic&quot;;
	padding: 0 8px;

	color:rgb(10,100,180);
border: 1px solid black;
border-color: rgb(165, 165, 165);
}</string>
   </property>
   <property name="text">
    <string>release year</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="releaseYear">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>530</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
	padding: 0 8px;
	background:white;
	border:10px;
	color:rgb(10,100,180);
border: 1px solid black;
border-color: rgb(165, 165, 165);
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="musicUrl">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

	border: 1px solid black;
	padding: 0 8px;
	background:;
	border-color: rgb(165, 165, 165);
	color:rgb(10,100,180);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="musicImg">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

	border: 1px solid black;
	padding: 0 8px;
	background:white;
	border-color: rgb(165, 165, 165);
	color:rgb(10,100,180);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label</zorder>
  <zorder>musicName</zorder>
  <zorder>label_6</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>category</zorder>
  <zorder>label_4</zorder>
  <zorder>releaseYear</zorder>
  <zorder>ratting</zorder>
  <zorder>label_5</zorder>
  <zorder>btnAddMusic</zorder>
  <zorder>statusInsert</zorder>
  <zorder>btnCancel</zorder>
  <zorder>label_9</zorder>
  <zorder>errorMusicImg</zorder>
  <zorder>errorMusicUrl</zorder>
  <zorder>errorMusicName</zorder>
  <zorder>musicUrl</zorder>
  <zorder>btnMusicUrl</zorder>
  <zorder>musicImg</zorder>
  <zorder>btnImgUrl</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
